spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/repository/NoSuchTaskExecutionException.java

```diff
@@ -16,13 +16,22 @@
 
 package org.springframework.cloud.dataflow.server.repository;
 
+import java.util.Set;
+
 import org.springframework.cloud.task.repository.TaskExecution;
+import org.springframework.util.StringUtils;
 
 /**
  * @author Glenn Renfro
+ * @author Gunnar Hillert
  */
 public class NoSuchTaskExecutionException extends RuntimeException {
 
+	/**
+	 *
+	 */
+	private static final long serialVersionUID = 1L;
```

Comment:
Remove empty javadoc and do a generated version instead of just `1L`.

Reply:
Fixed.